refaster-support/src/main/java/tech/picnic/errorprone/refaster/annotation/TypeMigration.java

```diff
@@ -0,0 +1,37 @@
+package tech.picnic.errorprone.refaster.annotation;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Indicates that a Refaster rule or group of Refaster rules is intended to migrate away from the
+ * indicated type.
+ */
+// XXX: Add support for `#unmigratedFields()`.
+// XXX: Consider making this annotation `@Repeatable`, for cases where a single Refaster rules
+// collection migrates away from multiple types.
+@Target(ElementType.TYPE)
+@Retention(RetentionPolicy.SOURCE)
```

Comment:
I agree that we don't lose much, but the current variant does seem a bit clearer to me. Since this annotation will only be present on a separate line in (almost always) small files, I don't think there's much point in forcing a static import here. Internally only 2 out of 67 usages statically import it.